src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,170 @@
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import FilterListIcon from '@material-ui/icons/FilterList';
+import BugsTableHead from './BugsTableHead';
+import BugsTableEntry from './BugsTableEntry';
+import { PROJECTS_PAGE_SIZE } from '../../utils/constants';
+
+const toolbarStyles = theme => ({
+  root: {
+    paddingRight: theme.spacing.unit,
+  },
+  spacer: {
+    flex: '1 1 100%',
+  },
+  actions: {
+    color: theme.palette.text.secondary,
+  },
+  title: {
+    flex: '0 0 auto',
+  },
+});
+let BugsTableToolbar = props => {
+  const { classes } = props;
+
+  return (
+    <Toolbar className={classes.root}>
+      <div className={classes.title}>
+        <Typography variant="title" id="tableTitle">
+          Bugs
+        </Typography>
+      </div>
+      <div className={classes.spacer} />
+      <div className={classes.actions}>
+        <Tooltip title="Filter list">
+          <IconButton aria-label="Filter list">
+            <FilterListIcon />
+          </IconButton>
+        </Tooltip>
+      </div>
+    </Toolbar>
+  );
+};
+
+BugsTableToolbar.propTypes = {
+  classes: PropTypes.object.isRequired,
+  numSelected: PropTypes.number.isRequired,
+};
+
+BugsTableToolbar = withStyles(toolbarStyles)(BugsTableToolbar);
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    marginTop: theme.spacing.unit * 3,
+  },
+  table: {
+    minWidth: 1020,
+  },
+  tableWrapper: {
+    overflowX: 'auto',
+  },
+});
+
+class BugsTable extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      order: 'desc',
+      orderBy: 'lastupdate',
+      page: 0,
+      rowsPerPage: PROJECTS_PAGE_SIZE,
+    };
+  }
+
+  handleRequestSort = (event, property) => {
+    const orderBy = property;
+    let order = 'desc';
+
+    if (this.state.orderBy === property && this.state.order === 'desc') {
+      order = 'asc';
+    }
+
+    this.setState({ order, orderBy });
+  };
+
+  handleChangePage = (event, page) => {
+    this.setState({ page });
+  };
+
+  render() {
+    const { classes, data, loading, error } = this.props;
+    const { order, orderBy, rowsPerPage, page } = this.state;
+    const emptyRows =
+      rowsPerPage - Math.min(rowsPerPage, data.length - page * rowsPerPage);
+
+    return (
+      <Fragment>
+        {loading && <div>loading...</div>}
+        {error ? (
+          <div>error...</div>
```

Comment:
We should output the error here. If you look in taskcluster-web, there is an `ErrorPanel` component you can copy.